yolox/utils/logger.py

```diff
@@ -241,15 +244,63 @@ def _log_validation_set(self, val_dataset):
                 id = data_point[3]
                 img = np.transpose(img, (1, 2, 0))
                 img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+
+                if isinstance(id, torch.Tensor):
+                    id = id.item()
+
                 self.val_table.add_data(
-                    id.item(),
+                    id,
                     self.wandb.Image(img)
                 )
 
             self.val_artifact.add(self.val_table, "validation_images_table")
             self.run.use_artifact(self.val_artifact)
             self.val_artifact.wait()
 
+    def _convert_prediction_format(self, predictions):
+        image_wise_data = defaultdict(int)
+
+        for key, val in predictions.items():
+            img_id = key
+            # print(key, val)
+            # quit()
+            try:
+                bboxes = val[0]
+                cls = val[1]
+                scores = val[2]
+            except KeyError:
+                bboxes = val["bboxes"]
+                cls = val["categories"]
+                scores = val["scores"]
+
+            act_box = []
```

Comment:
what does `act` prefix mean here?

Reply:
This is for the actual bounding boxes i.e. list of bounding boxes after removing all the Nones.
Added a comment to explain this.